server/foodAPI/IFoodAPI.ts

```diff
@@ -0,0 +1,32 @@
+import IBaseFood from "../serverAPI/model/food/IBaseFood";
+import IFood from "../serverAPI/model/food/IFood";
+
+export default interface IFoodAPI {
+    /**
+     * Retrieves collection of food items that satisfy searching parameters.
+     * 
+     * @param parameters query parameters used for searching.
+     * - query - required parameter that defines the name of the Food Item (partial names are accepted).
+     * - number - optional parameter that defines max numbe of the results to be returned.
```

Comment:
that's the name of the parameter they use in spoonacular api :rofl: I can change it internally and then just map it to the "number" in SpoonacularAPI I guess.

Reply:
Ok I see; if we can use a better name internally that would be good :)

Reply:
I created an issue. This issue is actually already solved in the update-authentication branch.... Essentially I created maps where key is the names we want to use and value is the original name of the parameter. Essentially it solves naming problems because we can call them whatever we want.